test/webauthn/apis.test.js

```diff
@@ -46,104 +42,730 @@ describe(`apisFunctions: ${printPath("[test/webauthn/apis.test.js]")}`, function
         await cleanST();
     });
 
-    it("test registerOptionsAPI with default values", async function () {
-        const connectionURI = await startST();
-
-        STExpress.init({
-            supertokens: {
-                connectionURI,
-            },
-            appInfo: {
-                apiDomain: "api.supertokens.io",
-                appName: "SuperTokensplm",
-                websiteDomain: "supertokens.io",
-            },
-            recipeList: [WebAuthn.init()],
+    describe("[registerOptions]", function () {
+        it("test registerOptions with default values", async function () {
+            const connectionURI = await startST();
+
+            STExpress.init({
+                supertokens: {
+                    connectionURI,
+                },
+                appInfo: {
+                    apiDomain: "api.supertokens.io",
+                    appName: "SuperTokensplm",
+                    websiteDomain: "supertokens.io",
+                },
+                recipeList: [WebAuthn.init()],
+            });
+
+            // run test if current CDI version >= 2.11
+            if (!(await isCDIVersionCompatible("2.11"))) return;
```

Comment:
I don't think this is right, it should be something like 5.12?

Reply:
you can just add a TODO to update it, we don't want to skip it right now. I think it's going to be 5.3 (the current one is 5.2)

Reply:
done